indico/logging.yaml

```diff
@@ -0,0 +1,45 @@
+# This is the default indico logging configuration.
+# It uses the standard Python dictConfig syntax, but provides a few enhancements:
+# - The `filename` of a `logging.FileHandler` is relative to the configured LOG_DIR
+# - The SMTP connection data of a `logging.handlers.SMTPHandler` is taken from the
+#   indico config (this is disabled if a custom `mailhost` is set).
+#   `fromaddr`, `toaddrs` and `subject` are also set to useful default values, but
+#   these values may be overridden.
+# - The `indico.core.logger.FormattedSubjectSMTPHandler` handler allows formatting
+#   the email subject using a standard python format string (e.g. `%(message)s`).
+#   Besides that it behaves exactly like the normal `SMTPHandler`, including the
+#   Indico-specific default values mentioned above.  It also handles unicode
+#   properly so it is not recommended to use the default SMTPHandler as that one
+#   may silently drop some log messages which contain non-ascii characters.
+# - All log formaters may use `%(request_id)s` to get a random ID unique to each
+#   request which makes it easier to grep for all other log entries that belong to
+#   a specific request.
+# - Setting `append_request_info` to `true` in a formatter will append various
+#   information about the current request to the log message.  This will result in
+#   a multi-line message and is only recommended for errors; usually in a handler
+#   which sends emails.
+
+version: 1
+
```

Comment:
Any particular reason why use double break lines?

Reply:
Actually no, I've removed it